in-cluster/default.yaml

```diff
@@ -3,8 +3,11 @@ kind: SupportBundle
 metadata:
   name: default
 spec:
-  uri: https://raw.githubusercontent.com/replicatedhq/troubleshoot-specs/main/in-cluster/default.yaml
+  uri: https://raw.githubusercontent.com/replicatedhq/troubleshoot-specs/main/host/default.yaml
```

Comment:
this should still be `in-cluster` right?

Reply:
Good catch

Reply:
This PR has changes from another branch :(

git-foo failing me :(

Reply:
Fixed